workflows/templates/main_table.html

```
@@ -1,3 +1,4 @@
+  <a id="{ocp_key}"/>
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)